src/layouts/components/Editor/MenuBar.tsx

```diff
@@ -0,0 +1,190 @@
+      title: "Paragraph",
+      action: () => editor.chain().focus().setParagraph().run(),
+      isActive: () => editor.isActive("paragraph"),
+    },
+    {
+      icon: "list-unordered",
+      title: "Bullet List",
+      action: () => editor.chain().focus().toggleBulletList().run(),
+      isActive: () => editor.isActive("bulletList"),
+    },
+    {
+      icon: "list-ordered",
+      title: "Ordered List",
+      action: () => editor.chain().focus().toggleOrderedList().run(),
+      isActive: () => editor.isActive("orderedList"),
+    },
+    {
+      icon: "list-check-2",
+      title: "Task List",
+      action: () => editor.chain().focus().toggleTaskList().run(),
+      isActive: () => editor.isActive("taskList"),
+    },
+    {
+      icon: "code-box-line",
+      title: "Code Block",
+      action: () => editor.chain().focus().toggleCodeBlock().run(),
+      isActive: () => editor.isActive("codeBlock"),
+    },
+    {
+      type: "divider",
+    },
+    {
+      icon: "double-quotes-l",
+      title: "Blockquote",
+      action: () => editor.chain().focus().toggleBlockquote().run(),
+      isActive: () => editor.isActive("blockquote"),
+    },
+    {
+      icon: "separator",
+      title: "Horizontal Rule",
+      action: () => editor.chain().focus().setHorizontalRule().run(),
+    },
+    {
+      type: "divider",
+    },
+    {
+      icon: "text-wrap",
+      title: "Hard Break",
+      action: () => editor.chain().focus().setHardBreak().run(),
+    },
+    {
+      icon: "format-clear",
+      title: "Clear Format",
+      action: () => editor.chain().focus().clearNodes().unsetAllMarks().run(),
+    },
+    {
+      type: "divider",
+    },
+    {
+      icon: "arrow-go-back-line",
+      title: "Undo",
+      action: () => editor.chain().focus().undo().run(),
+    },
+    {
+      icon: "arrow-go-forward-line",
+      title: "Redo",
+      action: () => editor.chain().focus().redo().run(),
+    },
+    {
+      icon: "file-image-line",
+      title: "Add image",
+      action: () => showModal("images"),
+    },
+    {
+      icon: "file-pdf-line",
+      title: "Add file",
+      action: () => showModal("files"),
+    },
+    {
+      icon: "links-line",
+      title: "Add link",
+      isActive: () => editor.isActive("link"),
+      action: () => editor.chain().focus().unsetLink().run(),
+    },
+    {
+      icon: "table-line",
+      title: "Add table",
+      action: () =>
+        editor
+          .chain()
+          .focus()
+          // NOTE: Default to smallest multi table
+          .insertTable({ rows: 3, cols: 3, withHeaderRow: true })
+          .run(),
+    },
+  ]
+
+  return (
+    <HStack
+      bgColor="gray.50"
+      flex="0 0 auto"
+      flexWrap="wrap"
+      p="0.25rem"
+      borderBottom="1px solid"
+      borderColor="base.divider.strong"
+      borderTopRadius="0.25rem"
+    >
+      {items.map((item, index) => (
+        // eslint-disable-next-line react/no-array-index-key
```

Comment:
quick win to just use uuid here? we alr have the lib in our fe iirc

Reply:
sure